website/content/docs/drivers/docker.mdx

```diff
@@ -239,6 +239,11 @@ config {
   configuration on the host (which is outside the scope of Nomad). Valid values
   pre-docker 1.9 are `default`, `bridge`, `host`, `none`, or `container:name`.
 
+  The default `network_mode` for tasks that use [Connect] will be
+  `container:<name>`, where the name is the container name of the parent
+  container used to share network namespaces between tasks. You should not set
+  `network_mode` for Connect-enabled tasks.
```

Comment:
What happens now when it's set? Should we add some validation/warning logic in the docker driver too?

Reply:
Your task can't reach the Envoy proxy and can't be reached over the Envoy proxy. I do want to add a warning but it's one of those annoying scenarios where the enforcement will be all client-side because it's in the driver config. In any case, I want to do any code changes to improve that in a separate PR. Also, I do worry that I haven't considered some weird corner case where you'd _want_ to do that?

Reply:
I've opened https://github.com/hashicorp/nomad/issues/10725 for the driver change, just in case I don't get to it right away.